app/states/spaceState.js

```diff
@@ -67,8 +65,9 @@ const spaceState = {
     // if (Math.random() * 1000 <= 1) this.spawnLoot()
 
     playerObject.movePlayer()
-    itemCollision(playerObject, projectile, localState.loot)
-    enemyCollision(playerObject, projectile, graveyard, localState.loot)
+
+    itemCollision(playerObject, localState.loot)
+    enemyCollision(playerObject, graveyard)
```

Comment:
please add localState.loot again to be the last argument for enemyCollision. Without it, enemyCollision causes an error if the enemy dies.